.github/workflows/dev-stg-release.yml

```diff
@@ -19,28 +19,48 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
 
+      - name: Build with Gradle
+        env:
+          packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
+          packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
+          JAVA_OPTS: -DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true
+        run: |
+          ./gradlew build
+          
       - name: Ballerina Build
-        uses: ballerina-platform/ballerina-action/@master
+        uses: ballerina-platform/ballerina-action/@2201.0.3
```

Comment:
As discussed offline, lets use `2201.1.0` instead of `2201.0.3`

Reply:
Ack fixed in fc08a84ae3b62f98c8863f7b4b430da95533b776